travis_configure.sh

```diff
@@ -24,19 +24,12 @@ print_error_msg ()
 
 # Script which is run before the installation script is called
 #
-# For linux based builds docker-compose is used to set up the build environment
-#
 # For osx based builds homebrew is used to install required packages:
 #   - sed, (libtool), openssl, mysql
 # Returns 0
 before_install()
 {
     if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
-        # build images
-        docker-compose -f ./docker-compose.yml \
-            -f ./docker/docker-compose.$DIST_NAME.$DIST_VERSION.yml \
-            build
-
```

Comment:
Why is this moved?

Reply:
The intention is that `before_install` only installs dependencies on the **travis** build environment. 
Thus it allows us to do `before_script: skip` if it isn't needed to build the docker images. As is the case in the deploy stage where the deb packages are uploaded.

Reply:
Maybe explain that in the commit message? :)
(and maybe even a comment in the travis file, so people seeing it know why it's organized like it's organized)